dpbench/configs/bench_info/rodinia/pathfinder.toml

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[benchmark]
+name = "Pathfinder Computation"
+short_name = "pathfinder"
+relative_path = "pathfinder"
+module_name = "pathfinder"
+func_name = "pathfinder"
+kind = "microbenchmark"
+domain = "Shortest Path"
+input_args = [
+    "data",
+    "rows",
+    "cols",
+    "pyramid_height",
+    "result",
+]
+array_args = [
+    "data",
+    "result",
+]
+output_args = [
+    "result",
+]
+
+[benchmark.parameters.S]
+rows=10
+cols=10
+pyramid_height=1
+
+[benchmark.parameters.M16Gb]
```

Comment:
This one should be almost the same as M, but memory should fit in 16Gb. You can refer how much memory was used in report. If memory is not a bottle neck, then it should be the same as `M`